src/main/java/com/redhat/devtools/lsp4ij/client/features/LSPClientFeatures.java

```diff
@@ -1105,6 +1140,7 @@ public void setServerCapabilities(@NotNull ServerCapabilities serverCapabilities
 
     @Nullable
     public TextDocumentServerCapabilityRegistry<? extends TextDocumentRegistrationOptions> getCapabilityRegistry(String method) {
+        // TODO: Can these be changed to else/if? Or can this be changed to switch?

```

Comment:
I don't know how often this methods gets called, but it looks like it can be changed to either `if ... else if ... else if ...` or ideally `switch (method)` to make it more efficient. I'm happy to do that as part of these changes if you'd like.

Reply:
Thanks so much to have done this horrible change!